### PR TITLE
Docs: Fix wrong word for tempo example config

### PR DESCRIPTION
**Pull request by wissyhost**
Replace traceToProfiles with tracesToProfiles

### CODE REVIEW

**Comment by knylander-grafana** [comment]
@wissyhost Thank you for the doc update. I'm curious what prompted your change to the configuration file. Did you encounter an issue? Did you test the fix you have submitted?

File: docs/sources/datasources/tempo/configure-tempo-data-source.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ datasources:
         queries:
           - name: 'Sample query'
             query: 'sum(rate(traces_spanmetrics_latency_bucket{$$__tags}[5m]))'
-      traceToProfiles:
+      tracesToProfiles:
```
Comment (knylander-grafana) [inline docs/sources/datasources/tempo/configure-tempo-data-source.md:250]: @grafana/observability-traces-and-profiling Is this update correct?

Comment (joey-grafana) [reply docs/sources/datasources/tempo/configure-tempo-data-source.md:250]: It's correct

**Review by knylander-grafana — APPROVED** [review]
Approved. Thank you for fixing this!

**Comment by jdbaldry** [comment]
@knylander-grafana does this need to be backported? A similar PR https://github.com/grafana/grafana/pull/84986 intended to backport to v10.4.x too.

**Comment by knylander-grafana** [comment]
> @knylander-grafana does this need to be backported? A similar PR #84986 intended to backport to v10.4.x too.

Yes, this would need to be babckported.

**Pull request merged**